install/kubernetes/cilium/values.yaml

```diff
@@ -206,7 +206,6 @@ azure:
   # Note that this is incompatible with AKS clusters created in BYOCNI mode: use
   # AKS BYOCNI integration (`aksbyocni.enabled`) instead.
   enabled: false
-  # usePrimaryAddress: false
```

Comment:
unrelated?

Reply:
it was because of "make -C install/kubernetes cilium/values.yaml"
one of the tests was failing without it